packaging/datadog-agent/source/install_agent.sh

```diff
@@ -73,15 +73,18 @@ Please use the 1-step script available at https://app.datadoghq.com/account/sett
 
 elif [ -f /etc/debian_version -o "$DISTRIBUTION" == "Debian" -o "$DISTRIBUTION" == "Ubuntu" ]; then
     OS="Debian"
-elif [ -f /etc/redhat-release -o "$DISTRIBUTION" == "RedHat" -o "$DISTRIBUTION" == "CentOS" -o "$DISTRIBUTION" == "openSUSE" -o "$DISTRIBUTION" == "Amazon" ]; then
+elif [ -f /etc/redhat-release -o "$DISTRIBUTION" == "RedHat" -o "$DISTRIBUTION" == "CentOS" -o "$DISTRIBUTION" == "Amazon" ]; then
     OS="RedHat"
 # Some newer distros like Amazon may not have a redhat-release file
 elif [ -f /etc/system-release -o "$DISTRIBUTION" == "Amazon" ]; then
     OS="RedHat"
 # Arista is based off of Fedora14/18 but do not have /etc/redhat-release
 elif [ -f /etc/Eos-release -o "$DISTRIBUTION" == "Arista" ]; then
     OS="RedHat"
-elif [ "$DISTRIBUTION" == "SUSE" ]; then
+# openSUSE and SUSE use /etc/SuSE-release (legacy), in the future it will be /etc/os-release
+elif [ -f /etc/os-release -o "$DISTRIBUTION" == "SUSE" -o "$DISTRIBUTION" == "openSUSE" ]; then
```

Comment:
@gmmeyer 

It looks like os-release is part of systemd, so yes it's very likely this will be present on other Linux distributions.

Source: [freedesktop.org:osrelease](https://www.freedesktop.org/software/systemd/man/os-release.html)

<pre>NAME=

    A string identifying the operating system, without a version component, and suitable for presentation to the user. If not set, defaults to "NAME=Linux". Example: "NAME=Fedora" or "NAME="Debian GNU/Linux"". </pre>